src/app/cli/src/tests/coda_restarts_and_txns_holy_grail.ml

```diff
@@ -12,39 +12,43 @@ let main n () =
     @@ Some
          (List.nth_exn Genesis_ledger.accounts 5 |> snd |> Account.public_key)
   in
-  let proposers n = if n < 3 then Some n else None in
   let%bind testnet =
-    Coda_worker_testnet.test logger n proposers snark_work_public_keys
+    Coda_worker_testnet.test logger n Option.some snark_work_public_keys
```

Comment:
Why does this even compile? In all the other calls this is an `int option`, but here it's `'a -> 'a option`??